azurerm/resource_arm_postgresql_configuration.go

```diff
@@ -0,0 +1,142 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/arm/postgresql"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/jen20/riviera/azure"
+)
+
+func resourceArmPostgreSQLConfiguration() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmPostgreSQLConfigurationCreateUpdate,
+		Read:   resourceArmPostgreSQLConfigurationRead,
+		Delete: resourceArmPostgreSQLConfigurationDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"resource_group_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"server_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"value": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
judging by the API Schema - I think we should be able to get away with a string for all of these values: 
<img width="1075" alt="screen shot 2017-08-04 at 11 54 08" src="https://user-images.githubusercontent.com/666005/28964457-195dca0e-790d-11e7-9aa5-935c5ebb3820.png">